### ISSUE TITLE
Implement sending ControllerRebootState message via nanopb

### ISSUE THREAD

**Issue by Miceuz** [RespiraWorks/VentilatorSoftware#126]
**Summary:**
Add nanopb implementation of ControllerIdentification message that is being sent on reboot currently.

**Details:**
Add serializing functionality and use it in comms library.

**Comment by jkff** [comment]
@jlebar Please add some details

**Comment by jlebar** [comment]
@Miceuz can you clarify what you meant by this bug?  Is this "send software version from controller to the GUI"?

**Comment by Miceuz** [comment]
Yes, now this issue is just about providing version information to GUI. 

Both firmware version and hardware version. I have asked electronics guys to include hardware identification via resistor combination on board, didn't yet look how they implemented that, but have seen the issue close.

**Comment by martukas** [comment]
need solution for how to set hardware version, but the SW version part of this can be picked up and implemented by soemone

**Comment by inceptionev** [comment]
Hardware Revision ID bits are on GPIO pins, described in the Rev A Interface Control Document lines 14 and 15: https://docs.google.com/spreadsheets/d/1EPEkAWZGx5ASWL2mgTzymS2f11Ow9ptx9c8Sycb3_Go/edit?usp=sharing

ID 00 is Rev 1.0, ID 01 is Rev A Hardware
